include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -142,6 +142,9 @@ extern "C" {
 // __VA_ARGS__ on Windows and Linux are different
 #define ORT_API(RETURN_TYPE, NAME, ...) RETURN_TYPE ORT_API_CALL NAME(__VA_ARGS__) NO_EXCEPTION
 
+#define ORT_API_T(RETURN_TYPE, NAME, ...) \
+  RETURN_TYPE(ORT_API_CALL* NAME)(__VA_ARGS__) NO_EXCEPTION
+
```

Comment:
We were missing the NO_EXCEPTION on a number of functions in the initial plugin EP setup. This macro should help keep things consistent by adding NO_EXCEPTION like ORT_API2_STATUS does.